modules/platforms/python/tests/test_sql.py

```diff
@@ -152,3 +152,71 @@ def test_sql_fields(client):
     # repeat cleanup
     result = sql_fields(client, 'PUBLIC', drop_query, page_size)
     assert result.status == 0
+
+
+def test_long_multipage_query(client):
+    """
+    The test creates a table with 13 columns (id and 12 enumerated columns)
+    and 20 records with id in range from 1 to 20. Values of enumerated columns
+    are = column number * id.
+
+    The goal is to ensure that all the values are selected in a right order.
+    """
+
+    field_range = range(1, 13)
+
+    record_range = range(1, 21)
+
+    drop_query = 'DROP TABLE LongMultipageQuery IF EXISTS'
+
+    create_query = '''CREATE TABLE LongMultipageQuery (
```

Comment:
Both of these queries can be easily created without so long definition:

i.e.
fields = ["id","abc"...]
```
"CREATE TABLE LongMultiPageQuery (%s, %s)" % (fields[0] +  " INT(11) PRIMARY KEY", ",".join(map(lambda f: f + " INT(11)", fields[1:])

"INSERT INTO LongMultipageQuery (%s) VALUES (%s)" % (",".join(fields), ",".join("?" * len(fields)))
```